NDB.Covid19/NDB.Covid19.iOS/Views/DailyNumbers/DailyNumbersViewController.cs

```diff
@@ -174,5 +175,10 @@ private void SetupSubTextWithLink(string text, UITextView textView)
 			//ForegroundColor sets the color of the links. UnderlineStyle determins if the link is underlined, 0 without underline 1 with underline.
 			textView.WeakLinkTextAttributes = new NSDictionary(UIStringAttributeKey.ForegroundColor, ColorHelper.TEXT_COLOR_ON_BACKGROUND, UIStringAttributeKey.UnderlineStyle, new NSNumber(1));
 		}
+
+		private void OnAppReturnsFromBackground(object o)
+		{
+			RequestFHIDataUpdate(SetStyling);
```

Comment:
Please ensure that the update of the UI happens on Main thread to prevent crashes.